lib/stripe/config.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Stripe.Config do
+  @moduledoc """
+  Utility that handles interaction with the application's configuration
+  """
```

Comment:
I recently added `json_library` config.  Would you mind pulling that into here?

Reply:
I rebaised to code-corps/stripity_stripe/master in `8562aca` and converted it to use the `Config` module